Jenkinsfile.private

```diff
@@ -0,0 +1,30 @@
+pipeline {
+	agent { node { label 'lisky' } }
+	stages {
+		stage('Prepare workspace') {
+			steps {
+				checkout scm
+			}
+		}
+		stage('Install dependencies') {
```

Comment:
Do we need this stage?

Reply:
Yes, snyk scans the `node_modules/` directory.